server/routes/api/chart.py

```diff
@@ -46,16 +46,16 @@
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_choropleth_places(geoDcid):
-    """ Get the list of places to show on a choropleth chart for a given place
+def get_choropleth_display_level(geoDcid):
```

Comment:
done. I think in this case, we want something general because the geoDcid isn't necessarily a parent or child, it's just the dcid of the place we're trying to get this information for.